### ISSUE TITLE
Ammo

### ISSUE THREAD

**Issue by Malkalm** [lekousin/lekousin-s-Armor-Overhaul#2]
It doesn't matter what armor i equip, it gives me ammo -skill 45  depends on weapon .

**Comment by Pecomare** [comment]
I don't understand what you're saying.
You get ammo bonus regardless the armor you equip, ok, but what do you mean by skill 45 depends on weapon" ?

**Comment by Malkalm** [comment]
Sorry my english is bad, nevermind it, i have solved it already. I was saying that every armor gave me -45 ammo.

**Comment by Pecomare** [comment]
Good to know your problem has been solved.
Do you know what happened and what you did ? It can help me if it's a flaw in the mod, because only Two-Piece Suit and Thin Suit should have ammo penalty.

**Comment by Malkalm** [comment]
When i instaled mod everything was ok,but when i dropped my skill tree that i wanted to re-do it gave me - ammo. Well i solved it that i just changed ammo value for suit and thin suit to original in upgradestweakdata.lua

**Comment by Pecomare** [comment]
That - ammo is intended.
Wearing a Two-piece Suit or a Thin Suit were changed so you have an ammo penalty wearing them.
The only thing is that those penalties are shown as skill penalties, that's all.

**Comment by Malkalm** [comment]
Yes i know that those two have ammo penalty,but every armor had those penalites when i dropped skill tree while i had equiped suit.
